application-mocca-calendar-ui/src/main/resources/MoccaCalendar/MoccaCalendarEventSheet.xml

```diff
@@ -263,21 +263,34 @@ $doc.display($field)
     : $doc.display('recurrent')
     #end
   #else
+  {{html xwiki='false'}}
     #if($!{request.readOnly} != "true")
```

Comment:
This is more of a question, I'm not that familiar with the code. For what type of events is this branch (that has `wiki=false`) reached? Is this reachable only for the events described by this ticket (imported ones) ? Or also for events created inside the XWiki Calendar app?
A comments would be helpful to describe on what cases this branch is reached and why you need the `wiki false`, since I know `wiki=true` is defined above

Reply:
Hello! Yes, this branch is only reached by events from iCals and gCals. The "xwiki false" setting is necessary because we don't have control over the data from other providers. If the wiki syntax is set to true, the data might be interpreted in a strange way and break the display.

Reply:
Ok, so it is indeed only for external calendars. It would be strange for them to have xwiki syntax :)  I would add these details as a comment in the code

Reply:
I updated the code with a comment